contracts/EscrowAndStaking/StakingThales.sol

```diff
@@ -128,6 +129,12 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
     IStakingThalesBonusRewardsManager public stakingThalesBonusRewardsManager;
```

Comment:
add to addressMAnager and use it from there. Make this variable private.

Reply:
won't be added since it's used in `updateVolume()`